.github/workflows/ci.yml

```diff
@@ -180,13 +180,21 @@ jobs:
       working-directory: ${{github.workspace}}/install
       run: ./f3d.app/Contents/MacOS/f3d ../source/testing/data/suzanne.obj --output=../output.png
 
+    # TODO: This job should be replaced by a useful example using the libf3d API and merged with the next job
     - name: Check SDK
       if: matrix.os != 'macos-latest'
       working-directory: ${{github.workspace}}
       run: |
         cmake -Werror=dev -Werror=deprecated --warn-uninitialized -Df3d_DIR:PATH=$(pwd)/install/lib/cmake/f3d -B test_sdk source/.github/test_sdk
         cmake --build test_sdk
 
+    - name: Build examples
+      if: matrix.os != 'macos-latest'
+      working-directory: ${{github.workspace}}
+      run: |
+        cmake -Werror=dev -Werror=deprecated --warn-uninitialized -Df3d_DIR:PATH=$(pwd)/install/lib/cmake/f3d -DVTK_DIR:PATH=$(pwd)/../dependencies/vtk_install/lib/cmake/vtk -B build_plugin source/examples/plugin
```

Comment:
I think plugin examples and libf3d SDK examples should be separated in two projects, as only one require VTK.

Reply:
indeed